app/Http/Controllers/GroupCreateController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Group;
+use App\GroupMember;
+
+use Illuminate\Http\Request;
+
+class GroupCreateController extends Controller
+{
+    //get friend list
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $req
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $req)
+    {
+        $this->validate($req, [
+            'name' => 'required|unique:posts|max:255' //don't know what posts do and the max value is temporary
```

Comment:
The example i gave you yesterday was just an example. Don't use unique posts. 

"unique:posts" means that the name must be unique in the "Posts" table in the database.

In your case, you should put "unique:groups"